vignettes/projecting_incidence.Rmd

```diff
@@ -196,6 +187,7 @@ chain_threshold <- 1000
 1. Reporting rate is constant through the course of the epidemic
 1. No interventions have been implemented
 1. Population is homogeneous and well-mixed
+1. All cases start chains of transmission that proceed independently and without competition for or depletion of susceptibles. This implies that none of the cases seen so far have infected each other.
```

Comment:
The last sentence might be quite a strong assumption here - and possibly be the reason why the median line ends up quite a bit above the data for the later recent dates. It might be preferable to use a shorter initial period to start the chains from, or to try and identify introductions if possible.

Reply:
That's interesting. Thanks for pointing it out.